app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java

```diff
@@ -108,149 +154,198 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+    }
+
+    @Override
+    public void onStartTrackingTouch(SeekBar seekBar) {
+
+    }
+
+    @Override
+    public void onStopTrackingTouch(SeekBar seekBar) {
+        _countColumns = (seekBar.getProgress() + 1) * 5;
+        _countRows = _countColumns * _height / _width;
+        _tvRowsAndColumns.setText(String.valueOf(_countColumns));
+        _tvRowsAndColumns.append(", ");
+        _tvRowsAndColumns.append(String.valueOf(_countRows));
+        showImage();
+        Panel panel = _viewColumnsFragment.findViewById(R.id.panel);
+        panel.doDraw();
+    }
+
+    /**
+     * Compare view rectangle with bitmap rectangle
+     *
+     * @return true if views length ratio bigger than bitmaps
+     */
+    private boolean isScreenWidth() {
+        double coefView = _imageViewWidth *1.0/(_imageViewHeight *1.0);
+        double coefBitmap = _bitmap.getWidth()*1.0/(_bitmap.getHeight()*1.0);
+        return !(coefView > coefBitmap);
+    }
+
+    /**
+     * Compute sizes of drawing grid
+     */
+    private void computeSizes() {
+        if(_isScreenWidth) {
+            _width =
+                    _imageViewWidth;
+            _propCoefficient =
+                    _width*1.0/(_bitmap.getWidth()*1.0);
+            _height =
+                    (int)(_bitmap.getHeight() * _propCoefficient);
+            _startHeight =
+                    (_imageViewHeight - _height)/2;
+            _startWidth =
+                    0;
+        } else {
+            _height =
+                    _imageViewHeight;
+            _propCoefficient =
+                    (_height)*1.0/(_bitmap.getHeight()*1.0);
+            _width =
+                    (int)(_bitmap.getWidth() * _propCoefficient);
+            _startHeight =
+                    0;
+            _startWidth =
+                    (_imageViewWidth - _width)/2;
+        }
+    }
+
     /**
      * Compute sizes in onCreateView
      *
      * @param is_saved true if SavedInstance != null
      */
-    public void computeSizes(boolean is_saved) {
+    private void setSizes(boolean is_saved) {
         Context context = getContext();
+        if (context == null) return;
         WindowManager wm = (WindowManager) context.getSystemService(Context.WINDOW_SERVICE);
         Display display = wm.getDefaultDisplay();
 
         DisplayMetrics metrics = new DisplayMetrics();
         display.getMetrics(metrics);
 
         int resourceId = getResources()
-                .getIdentifier("status_bar_height", "dimen", "android");
+                .getIdentifier(_IDENTIFIER_NAME, _IDENTIFIER_DEF_TYPE, _IDENTIFIER_DEF_PACKAGE);
         int status_bar_height = getResources().getDimensionPixelSize(resourceId);
 
         Point size = new Point();
         display.getSize(size);
-        int screen_width = size.x;
-        int screen_height = size.y;
+        int screenWidth = size.x;
+        int screenHeight = size.y;
 
         TypedArray styledAttributes = getContext().getTheme().obtainStyledAttributes(
```

Comment:
This line has a length over than 80. Please check [Method chain case](https://github.com/nonoteam/NonoConverter/blob/master/CodingStandard.md#line-wrapping-strategies) in CodingStandard.md

Reply:
Please check